sdk/core/azure-core/src/main/java/com/azure/core/credential/SimpleTokenCache.java

```diff
@@ -64,8 +64,8 @@
  */
 public class SimpleTokenCache {
     // The delay after a refresh to attempt another token refresh
-    private static final Duration REFRESH_DELAY = Duration.ofSeconds(30);
-    private static final String REFRESH_DELAY_STRING = String.valueOf(REFRESH_DELAY.getSeconds());
+    private static Duration refreshDelay = Duration.ofSeconds(30);
+    private static final String REFRESH_DELAY_STRING = String.valueOf(refreshDelay.getSeconds());
```

Comment:
If `refreshDelay` is going to be mutable, I think we should remove this constant String as this could become wrong.